public/content/documentation/web/page-level/form.md

```diff
@@ -1,80 +1,603 @@
+    </div>
+
+    <label for="phone">
+      Phone number
+    </label>
+    <input  type="tel"
+            id="phone"
+            inputmode="numeric"
+            autocomplete="tel"
+            aria-describedby="hint-phone">
+    <div class="hint" id="hint-phone">
+      Format: 573-268-9692
+    </div>
+
+    <label for="message">
+      Your message
+    </label>
+    <textarea id="message"></textarea> 
+      
+
+  </fieldset>
+</form>
+```
+<example>
+    <form aria-label="Contact us">
+          <fieldset>
+            <legend>
+              Preferred contact method
+            </legend>
+            <input type="radio" name="method" id="contact-email" checked>
+            <label for="contact-email">Email</label>
+            <input type="radio" name="method" id="contact-sms">
+            <label for="contact-sms">SMS text</label>
+            <input type="radio" name="method" id="contact-phone">
+            <label for="contact-phone">Phone</label>
+          </fieldset>
+          <fieldset>
+            <legend>
+              Your information
+            </legend>
+            <label for="email">
+              Email address
+            </label>
+            <input id="email"
+                   type="email"
+                   autocomplete="email"
+                   spellcheck="false"
+                   aria-describedby="hint-email">
+            <div class="hint" id="hint-email">
+              We’ll never sell or share your information
+            </div>
+            <label for="phone">
+              Phone number
+            </label>
+            <input  type="tel"
+                    id="phone"
+                    inputmode="numeric"
+                    autocomplete="tel"
+                    aria-describedby="hint-phone">
+            <div class="hint" id="hint-phone">
+              Format: 573-268-9692
+            </div>
+            <label for="message">
+              Your message
+            </label>
+            <textarea id="message"></textarea>
+          </fieldset>
+        </form>
+</example>
+
+## Output from inputs
+- Screenreader support varies output can be used for a dynamic content that changes based on user inputs (example: a calculator).
+- Alternatively, using a custom element with role=”status” will achieve more predictable results
+
+```html
+<form oninput="result.value=parseInt(rent.value)+parseInt(utilities.value)">
+  <fieldset>
+    <legend>
+      Calculate monthly expenses
+    </legend>
+    
+    <label for="rent">Monthly rent</label>
+    <input type="text" id="rent" inputmode="numeric" value="2500" maxlength="4">
+
+    <label for="utilities">Monthly utilities</label>
+    <input type="text" id="utilities" inputmode="numeric" value="500" maxlength="4">
+    
+    <label for="result">Total</label>
+    <input 
+      readonly 
+      type="text"
+      id="result"
+      inputmode="numeric" 
+      value="3000" 
+      maxlength="6">
+  </fieldset>
+</form>
+```
 
```

Comment:
Include ## Further reading with the following resource:

- `[WebAIM Accessible Form validation and error recovery](https://webaim.org/techniques/formvalidation/)`